efficientdet/hparams_config.py

```diff
@@ -183,6 +186,7 @@ def default_detection_configs():
   h.autoaugment_policy = None
   h.grid_mask = False
   h.sample_image = None
+  h.map_freq = 5
```

Comment:
Maybe add a comment like:

    h.map_freq = 5     # AP eval frequency in epochs.